stable/mongodb-replicaset/mongodb-up-test.sh

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env bash
+
+    pod_name() {
+        local full_name="${FULL_NAME?Environment variable FULL_NAME not set}"
+        local index="$1"
+        echo "$full_name-$index.$full_name.default.svc.cluster.local"
```

Comment:
This assumes `default` namespace, and a certain cluster domain: `svc.cluster.local`, which recently tripped people up.
It is a test, but I'm not sure that we should make this assumption.